### ISSUE TITLE
Use `sh` instead of `bash`

### ISSUE THREAD

**Issue by modulo11** [paketo-buildpacks/apache-tomcat#324]
Currently, the `catalina.sh` script is called using `bash`, although the script includes a `#!/bin/sh` shebang. Was this a conscious choice and what was the reason for that?

We think that using a "smaller" shell (like dash) can eventually lead to smaller images with smaller attack surface.

## Describe the Enhancement

If appropriate, the shell used in https://github.com/paketo-buildpacks/apache-tomcat/blob/42522ab83acb7ab16490760316d6a954dcf2f1a3/tomcat/build.go#L156 can be exchanged with `sh` instead of `bash`.

## Motivation

Allow smaller stacks without `bash` eventually.

**Comment by dmikusa** [comment]
I believe I asked about this previously and we've defaulted to using bash because there is a requirement in the spec [that bash be used if the spec calls for a shell](https://github.com/buildpacks/spec/blob/main/README.md?plain=1#L30), which generally means bash will be available (at least at build time). That assumption has probably spread more than is strictly necessary.

I can also say that of the Paketo stacks, tiny has no shell at all, and base and full both have bash. So you could change this but with the default selection of Paketo stacks it's not going to really matter. It won't allow running Tomcat on tiny more easily (we have some workarounds in place to run on the tiny stack). This change would probably help for getting the buildpack to run on more different stacks through.

Anyway, that's the context I have on it.

tl;dr I'm not opposed to making the change. I see your PRs and I'll try to take a look at them, possibly towards the end of the week.
